packages/real-hoist/test/index.ts

```diff
@@ -1,8 +1,11 @@
-import path from 'path'
-import hoist from '@pnpm/real-hoist'
-import { readWantedLockfile } from '@pnpm/lockfile-file'
+import { test, expect } from "vitest";
+import path from "path";
+import hoist from "@pnpm/real-hoist";
+import { readWantedLockfile } from "@pnpm/lockfile-file";
 
-test('hoist', async () => {
-  const lockfile = await readWantedLockfile(path.join(__dirname, '../../..'), { ignoreIncompatible: true })
-  expect(hoist(lockfile!)).toBeTruthy()
-})
+test("hoist", async () => {
+  const lockfile = await readWantedLockfile(path.join(__dirname, "../../.."), {
+    ignoreIncompatible: true,
+  });
+  expect(hoist(lockfile!)).toBeTruthy();
+});
```

Comment:
turn off prettier for pnpm. We currently use standardjs style.

Reply:
Done!